### ISSUE TITLE
Zoom in B/B Editor?

### ISSUE THREAD

**Issue by GingkathFox** [LMMS/lmms#4539]
I think it would be better to have the ability to zoom in the B/B editor. It would help with long loops, and it would be easier and quicker to make a beat.

**Comment by EtornaZ** [comment]
If you want to be able to place percussion easier in the B&B editor, just stretch the box out so that it's longer.

Example- before stretched
![image](https://user-images.githubusercontent.com/33297131/44158751-82c66480-a083-11e8-827e-9ff4be5e8ad0.png)

Example- after stretched
![image](https://user-images.githubusercontent.com/33297131/44158820-b99c7a80-a083-11e8-94c5-7e93297705da.png)

**Comment by GingkathFox** [comment]
I maximized it.
![screen shot 2018-08-15 at 2 35 23 pm](https://user-images.githubusercontent.com/33764485/44166074-7993c280-a098-11e8-93ec-d08600e40e6e.png)

**Comment by musikBear** [comment]
@GingkathFox You could open it in piano-roll (?)

**Comment by GingkathFox** [comment]
true, ~~Which is what I did~~ But it's a bit annoying, especially when you have rapid notes.
